demo-app/src/app/card/card.component.html

```diff
@@ -1 +1,55 @@
-<p>card works!</p>
+<div class="card-box container">
+  <div fxLayout="row" fxLayoutAlign=" center center" fxLayoutGap="760px">
```

Comment:
How is this 760px used? Avoid having huge for layouts

Reply:
screenshot please